woocommerce/woocommerce
# Add line number reference links to release posts for hooks

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds the GitHub line number reference link to the hooks that have been added for the release posts. Aside, I also fixed an issue with the template filepath not showing up on the post.



- [] This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).



### How to test the changes in this Pull Request:



1. Checkout this branch.
2. `pnpm install`
3. `pnpm run build`
4.  `cd tools/release-posts`
5. Run `pnpm run release-post -- "6.9.0" --outputOnly`
6. Check the output HTML and ensure you see the GitHub links to the hooks in question and that it points correctly in GitHub.
7. Also check that you see template filepaths are showing up correctly.



### Other information:




### FOR PR REVIEWER ONLY:
